packages/client/src/generators.ts

```diff
@@ -21,18 +21,18 @@ function generateRandomString(length: number) {
  * Generates random bytes and encodes them in url safe base64.
  * @param length The length of the string (before base64url-encoding) to generate.
  */
-export const generateRandomInBase64 = (length = RANDOM_STRING_MAX_LENGTH) =>
-  encodeBase64(generateRandomString(length));
+export const generateRandomInUrlSafeBase64 = (length = RANDOM_STRING_MAX_LENGTH) =>
+  encodeInUrlSafeBase64(generateRandomString(length));
```

Comment:
just `encodeUrlSafeBase64` will do

Reply:
also is there a standard for us, which method should lies under `utils` which should remain in this file? looks to me `generateRandomString` & `encodeUrlSafeBase64` share a same use case. 
- both are only used in this file
- both are general string operators

either move all to `utils` or keep in `generators` works for me . Just need to align a standard

Reply:
- I prefer to put all `generate*()` methods in `generators.ts`.

    I will move `generateScope()` from `utils.ts` to `generators.ts` in another PR.